File: Baekjoon/Swift/Implementation/Q9093.swift
//
//  main.swift
//  BOJ_PS
//
//  Created by 김태훈 on 2021/05/06.
//  단어 뒤집기 / 문자열

let t = Int(readLine()!)!

for _ in 0 ..< t {
    let strs = readLine()!.split(separator: " ").map { String($0) }
    var reversedStrs = [String]()
    for str in strs {
        reversedStrs.append(String(str.reversed()))
    }
    print(reversedStrs.joined(separator: " "))
}

File: Baekjoon/Python/BOJ_PS/Implementation/Q9093.py
# 단어 뒤집기 / 문자열
import sys

input = sys.stdin.readline
n = int(input())

for _ in range(n):
    strs = input().split()
    for str in strs:
        print(''.join(reversed(list(str))), end=' ')
    print()
